pages/doc/wavefront_spring_boot_faq.md

```diff
@@ -31,6 +32,7 @@ Follow these steps:
   # Spring Boot 2:
   management.metrics.export.wavefront.api-token=44444-34this-45is-123a-sampletoken
   ```
+-->
```

Comment:
Same question.

Reply:
Yes, for this. Because in the future the freemium will be removed too. But, I wanted to be 100% sure before I removed it completely :) .. If everything goes as planned.. I will remove it when the freemium offering removal.